packages/editor/editor.css

```diff
@@ -1138,30 +1138,42 @@ input,
 
 /* -------------------- NoteShape ------------------- */
 
-.tl-note__container {
+.tl-note {
 	position: relative;
+}
```

Comment:
In order to support transparency, we need the "body, shadow, and scrim" to be inside of one div (tl-note__container) while the label (which is not transparent) is in another (tl-note).